### DISCUSSION TITLE
Running fwupd in a Kubernetes Pod

### DISCUSSION THREAD

**Discussion by dnoliver** [fwupd/fwupd#4020] [Q&A]
**Describe the question**
Is it possible to run the `fwupdmgr security --force` command from a pod in a k8s environment?
I know you need to get some services into systemd, and to actually run the security checks you will need to mount security sensitive volumes, and run it in privileged mode.

I wonder is somebody already tried this, and have a method to actually make it work.

The intention is to have a deamonset-like agent that periodically runs fwupd, withouth having to modify the host os to make this happen. For example, if you have a Fedora Core OS based installation, which restrict the packages that can be installed in the host, it may be good have a pod that can run the fwupd checks on the nodes in the cluster.  

**fwupd version information**
Please provide the version of the daemon and client if applicable to your current installation of `fwupd`.

Latest stable version in Fedora

```shell
fwupdmgr --version
```

**Comment by hughsie** [comment]
The short answer is I don't _think_ so. The longer answer is about a year ago we had to shuffle around the way we shipped fwupd for CoreOS, so I suspect someone was doing that then. Sorry that's not more helpful.
